client/debug/main.go

```diff
@@ -0,0 +1,178 @@
+package debug
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	"strconv"
```

Comment:
If you don't format this with `goimports` @alexanderbez will come and get you in your sleep

![image](https://user-images.githubusercontent.com/229356/67579013-8fd8ec00-f73b-11e9-8cd4-1e2de4064fef.png)

Reply:
++